### PR TITLE
feat(factory): add isUrd getter

### LINKED ISSUE #56: Add a mapping to the URDFactory?

### PR DESCRIPTION
**Pull request by Rubilmax**
- Fixes #56

### CODE REVIEW

**Review by MerlinEgalite — CHANGES_REQUESTED** [review]
Can be added to the tests

**Review by MerlinEgalite — APPROVED** [review]

File: src/UrdFactory.sol [hunk 1]
```diff
@@ -10,6 +10,12 @@ import {UniversalRewardsDistributor} from "./UniversalRewardsDistributor.sol";
 /// @custom:contact security@morpho.org
 /// @notice This contract allows to create UniversalRewardsDistributor (URD) contracts, and to index them easily.
 contract UrdFactory {
+    /* STORAGE */
+
+    mapping(address => bool) public isUrd;
```
Comment (julien-devatom) [inline src/UrdFactory.sol:15]: so we are not storing the creator (msg.sender) address

**Review by julien-devatom — APPROVED** [review]

File: src/UrdFactory.sol (hunk 1 above)
Comment (MerlinEgalite) [reply src/UrdFactory.sol:15]: No we came to the conclusion that it was not necessary as we can easily retrieve the address of the deployer and thus it adds unnecessary logic

Comment (Rubilmax) [reply src/UrdFactory.sol:15]: (which results in unnecessary complexity on Etherscan when using this getter)

Comment (Rubilmax) [reply src/UrdFactory.sol:15]: (or unnecessary complexity when using this getter on Etherscan)

Comment (julien-devatom) [reply src/UrdFactory.sol:15]: Clear!

**Pull request merged**